pyperformance/data-files/benchmarks/bm_random/run_benchmark.py

```diff
@@ -0,0 +1,55 @@
+"""Test the performance of random module.
+
+This benchmark was available as `python -m random --test` in Python 3.13.
+
+Authors: Hugo van Kemenade (original), Semyon Moroz (pyperformance port).
```

Comment:
I'm not the original author; it was Guido, 32 years ago!

https://github.com/python/cpython/commit/ff03b1ae5bba4d6712563efb7c77ace57dbe6788#diff-851cf2d8d99b6d8dee26acc31319431dce51dbe26e3e56494d9b4c45edd8e11fR177

```suggestion
Authors: Guido van Rossum (original), Semyon Moroz (pyperformance port).
```